keyboards/a_jazz/akc084/keyboard.json

```diff
@@ -0,0 +1,128 @@
+{
+    "manufacturer": "a-jazz",
+    "keyboard_name": "akc084",
+    "maintainer": "Feng",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A6", "pin_b": "A7"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "A9",
+        "num_lock": "A10"
+    },
+    "matrix_pins": {
+        "cols": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B11", "B12", "B13"],
+        "rows": ["A0", "A1", "A2", "A3", "A4", "A5", "A8"]
+    },
+    "processor": "STM32F103",
+    "usb": {
+        "device_version": "1.0.1",
+        "no_startup_check": true,
+        "pid": "0x0084",
+        "vid": "0x3151",
```

Comment:
This VID is already in use, and should be changed
https://www.the-sz.com/products/usbid/index.php?v=0x3151&p=&n=

Reply:
Hi, 0x28E9 is the vid of GDdevice.

Reply:
change VID 0X414A